Sources/StreamChat/APIClient/RequestEncoder.swift

```diff
@@ -288,6 +288,7 @@ private extension URL {
         // value for a space. We want to keep it as "+" so we have to manually percent-encode it.
         components.percentEncodedQuery = components.percentEncodedQuery?
             .replacingOccurrences(of: "+", with: "%2B")
+            .replacingOccurrences(of: ";", with: "%2B")
```

Comment:
I think we'll need a unit-test with that string you used with many other special character.

Reply:
`%3`

Reply:
Updated 👍